tests/contracts/equal.clar

```diff
@@ -130,6 +134,10 @@
     (ok (is-eq (err 0x123456) (err 0xabcdef)))
 )
 
+(define-public (call-response-some-err-same-value-unequal)
```

Comment:
```suggestion
(define-public (call-response-ok-err-same-value-unequal)
```

Reply:
Jesus....

Reply:
fixed in [67d812d](https://github.com/stacks-network/clarity-wasm/pull/174/commits/67d812d41cd70e7489c4a050bd1b84d5ba98aaca)